core/src/test/java/org/fao/geonet/kernel/security/openidconnect/bearer/UserInfoCacheTest.java

```diff
@@ -0,0 +1,32 @@
+package org.fao.geonet.kernel.security.openidconnect.bearer;
```

Comment:
I think this needs a (c) header.  Please grab one from another file.

Reply:
Otherwise - looks great and a with a test case.

Thanks!!  Ok to merge once you put a (c) header in.

Thanks for the great catch and PR!

Reply:
Done!